department-of-veterans-affairs/va.gov-cms
# VACMS-6950: Remove "Page last built" field from 3 content types

- Configuration changes: Removed the "field_page_last_built" field from the "page", "landing_page", and "support_service" content types. Deleted "field_page_last_built" field config files.
- Code changes: Scanned the "va_gov_build_trigger" custom module and removed references to the "field_page_last_built" field and its dependencies.
- Behat: Updated "content_model_benefits_hubs_content_type_fields.feature" behat test and removed all "field_page_last_built" field references.

- Blocked by content-build PR, waiting for merge and release to prod: https://github.com/department-of-veterans-affairs/content-build/pull/819

## Description

closes #6950

## Testing done

All build test passed.

## Screenshots


## QA steps

As user administrator

1. Go to `/admin/structure/types/manage/page/fields`
    - [x] Verify and confirm that the `field_page_last_built` have been removed from the "page" content type
    - [x] Create a "page" node (`node/add/page`) and verify and confirm there are no errors and all worked as expected
    - [x] Edit an existing "page" node and verify and confirm there are no errors and all worked as expected
    - [x] Check and confirm there are no errors: Go to `/admin/reports/dblog` and take a look at the latest reports

2. Go to `/admin/structure/types/manage/landing_page/fields`
    - [x] Verify and confirm that the `field_page_last_built` have been removed from the "landing_page" content type
    - [x] Create a "landing_page" node (`node/add/landing_page`) and verify and confirm there are no errors and all worked as expected
    - [x] Edit an existing "landing_page" node and verify and confirm there are no errors and all worked as expected
    - [x] Check and confirm there are no errors: Go to `/admin/reports/dblog` and take a look at the latest reports

3. Go to `/admin/structure/types/manage/support_service/fields`
    - [x] Verify and confirm that the `field_page_last_built` have been removed from the "support_service" content type
    - [x] Create a "support_service" node (`node/add/support_service`) and verify and confirm there are no errors and all worked as expected
    - [x] Edit an existing "support_service" node and verify and confirm there are no errors and all worked as expected
    - [x] Check and confirm there are no errors: Go to `/admin/reports/dblog` and take a look at the latest reports

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
